composer.json

```diff
@@ -11,7 +11,8 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "phergie/phergie-irc-bot-react": "~1"
+        "phergie/phergie-irc-bot-react": "~1",
+        "phergie/phergie-irc-parser": "~1.6"
```

Comment:
It should be `>=1.6` rather than `~1.6`.

It's to do with accessing a specific indexed parameter from a numeric message. The index numbering changed in v1.6, so the line `$primaryNick = $params[1];` wouldn't store the right parameter if parser <=1.6 were used.

It's pendantic, I know...